File: test/fixtures/output/java/unirest/application-form-encoded.java
//Import unirest libarary (http://unirest.io/java.html) 

HttpResponse<String> response = Unirest.post("http://mockbin.com/har")
  .header("content-type", "application/x-www-form-urlencoded")
  .body("foo=bar&hello=world")
  .asString();

File: test/fixtures/output/node/unirest/application-form-encoded.js
const unirest = require("unirest");

const req = unirest("POST", "https://httpbin.org/anything");

req.headers({
  "content-type": "application/x-www-form-urlencoded"
});

req.form({
  "foo": "bar",
  "hello": "world"
});

req.end(function (res) {
  if (res.error) throw new Error(res.error);

  console.log(res.body);
});

